### PR TITLE
feat(maintenance): add maintenance feature's router

### PR DESCRIPTION
**Pull request by theoatboy139**
- Added router for path `/maintenance`
- Added homepage for maintenance feature

### CODE REVIEW

File: src/views/maintenance/home.vue [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    <h1>MAINTENANCE HOME</h1>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue } from "vue-property-decorator";
+export default class MaintenanceHome extends Vue {}
+</script>
+
+<style scoped></style>
```
Comment (beesk135) [inline src/views/maintenance/home.vue]: not needed here?

File: src/views/maintenance/index.vue [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<template>
+  <router-view />
+</template>
+
+<script lang="ts">
+import { Vue } from "vue-property-decorator";
+export default class Maintenance extends Vue {}
+</script>
+
+<style scoped></style>
```
Comment (beesk135) [inline src/views/maintenance/index.vue]: not needed here?

File: src/views/maintenance/maintenance-router.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  component: Maintenance,
+  children: [
+    {
+      path: "",
+      component: MaintenanceHome
+    },
+    {
+      path: "*",
+      redirect: "/maintenance"
+    }
```
Comment (beesk135) [inline src/views/maintenance/maintenance-router.ts]: what is * path?

File: src/views/maintenance/index.vue (hunk 2 above)
Comment (theoatboy139) [reply src/views/maintenance/index.vue]: I will remove it

File: src/views/maintenance/home.vue (hunk 1 above)
Comment (theoatboy139) [reply src/views/maintenance/home.vue]: I will remove it

File: src/views/maintenance/maintenance-router.ts (hunk 3 above)
Comment (theoatboy139) [reply src/views/maintenance/maintenance-router.ts]: just like regex, it matches anything that is not matched all of above

**Pull request merged**